packages/backend-core/src/security/roles.ts

```diff
@@ -213,6 +213,22 @@ export function getBuiltinRole(roleId: string): Role | undefined {
   return cloneDeep(role)
 }
 
+export function validInherits(
+  allRoles: RoleDoc[],
+  inherits?: string | string[]
+): boolean {
+  if (!inherits) {
+    return false
+  }
+  const find = (id: string) => allRoles.find(r => compareRoleIds(r._id!, id))
```

Comment:
`compareRoleIds` naming makes this a bit tricky to understand what's going on at first glance. Would suggest a rename to `roleIdsAreEqual(r._id!, id)`